python/sglang/srt/managers/tokenizer_manager.py

```diff
@@ -1190,7 +1190,9 @@ def abort_request(self, rid: str = "", abort_all: bool = False):
     async def pause_generation(self):
         async with self.is_pause_cond:
             self.is_pause = True
-            self.abort_request(abort_all=True)
+            while self.model_update_lock.is_locked():
```

Comment:
nit: add a comment mentioning we are using "model update" lock not for its model update semantics, but for its "there is ongoing req" semantics

Reply:
fixed.